autogen.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/sh
 
-. ./fetch-zfs-branch.sh
+if [ ! -z ${TRAVIS_TAG} ];
+then
+  ZFS_BUILD_BRANCH=${TRAVIS_TAG}
```

Comment:
should we do an auto commit for fetch-zfs-branch.sh using git credentials, as part of future PRs?

Reply:
probably, this can be done along with the auto commit of tags creation during release branch creation time

Reply:
hmm. need to handle the case of RC tags vs actual release tag. Was thinking we should do it as part of the main release tag - branch creation stage.